tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": "1.6.2",
+  "compilerOptions": {
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "target": "es5",
+    "module": "commonjs",
+    "sourceMap": true
+  },
+  "files": [
+    "src/app/bootstrap.ts",
+    "src/typings/tsd.d.ts"
+  ]
+}
```

Comment:
it's sufficient to just mention `bootstrap.ts`, everything is linked from that one. And typings is not needed anymore either.